### DISCUSSION TITLE
Pressure Advance question

### DISCUSSION THREAD

**Discussion by Padawg** [OrcaSlicer/OrcaSlicer#107] [Q&A]
So I am use to my other printers and setting the pressure advance in firmware for the machine after running the test pattern.  Are we doing this test for each brand of filament or are we setting the same number in all our filaments.  I know flow is different per filament. 
I assume its one number but everything seems different with the X1C.

**Comment by SoftFever** [comment]
I personally don't do PA calibration for each brand. I do this for each filament type(ABS/PLA/TPU).
For flow rate, however, I test each brand.

**Comment by Matrekus** [comment]
By the way... I think you have a writing mistake in the files list.. only as remark.
![PA](https://user-images.githubusercontent.com/100630918/211163393-21f12c9c-b791-46ce-8fd6-a846ad278832.png)
